README.md

```diff
@@ -3,55 +3,52 @@
 [![Hits-of-Code](https://hitsofcode.com/github/GrayJack/coreutils)](https://hitsofcode.com/view/github/GrayJack/coreutils)
 [![Build Status](https://api.travis-ci.com/GrayJack/coreutils.svg?branch=master)](https://travis-ci.com/GrayJack/coreutils)
 [![FOSSA Status](https://app.fossa.io/api/projects/git%2Bgithub.com%2FGrayJack%2Fcoreutils.svg?type=shield)](https://app.fossa.io/projects/git%2Bgithub.com%2FGrayJack%2Fcoreutils?ref=badge_shield)
+<!-- [![FOSSA Status](https://app.fossa.io/api/projects/git%2Bgithub.com%2FGrayJack%2Fcoreutils.svg?type=large)](https://app.fossa.io/projects/git%2Bgithub.com%2FGrayJack%2Fcoreutils?ref=badge_large) -->
 
 
-An attempt to make a safe re-implementation coreutils aiming for a minimal and yet complete set of utilities. This project aims to have **at least** POSIX especification and the common features between several implementations of the utility.
-
-This project has no intent to be 100% compatible with GNU's coreutils, like [Uutils' coreutils](https://github.com/uutils/coreutils), but if happens to be, that's ok too.
+An in-progress safe implementation of `coreutils` aiming for a minimal and yet complete set of utilities. This project aims at the _POSIX_ specification basic requirements, plus common and useful features present in other implementations of the utility.
 
-## Contributing
-Check the [CONTRIBUTING.md](./CONTRIBUTING.md) file for the guidelines to contribute to the project, including issue reports, git commits messages, etc.
+This project has no intent to be compatible with _GNU's coreutils_, like [Uutils' coreutils](https://github.com/uutils/coreutils).
 
 ## Minimum Rust Version Policy
-This project's minimum supported `rustc` version (MSRV) is 1.40.0.
+This project's minimum supported `rustc` version (_MSRV_) is _1.40.0_.
```

Comment:
1.43.0 is the MSRV

Reply:
So, it should be replaced by

"This project's minimum supported `rustc` version (_MSRV_) is _1.43.0_."

?